src/Component/src/Reflection/ClassReflection.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Resource\Reflection;
+
+use Symfony\Component\Finder\Finder;
+
+final class ClassReflection
+{
+    public static function getResourcesByPaths(array $paths): iterable
+    {
+        foreach ($paths as $resourceDirectory) {
+            $resources = self::getResourcesByPath($resourceDirectory);
+
+            foreach ($resources as $className) {
+                yield $className;
+            }
+        }
+    }
+
+    public static function getResourcesByPath(string $path): iterable
+    {
+        $finder = new Finder();
+        $finder->files()->in($path)->name('*.php')->sortByName(true);
+
+        foreach ($finder as $file) {
+            $fileContent = (string) file_get_contents((string) $file->getRealPath());
```

Comment:
```suggestion
            $fileContent = file_get_contents((string) $file->getRealPath());
```

`file_get_contents` might return **false**, so it might be better to handle it explicitly.

Reply:
Done